File: Python/CS50P/grade.py
score = int(input("Score: "))

if 90 <= score <= 100: # score >= 90:
    print("Grade: A")
elif 80 <= score < 90: # score >= 80:
    print("Grade: B")
elif 70 <= score < 80: # score >= 70:
    print("Grade: C")
elif 60 <= score < 70: # score >= 60:
    print("Grade: D")
else:
    print("Grade: F")

File: Python/CS50P/C/grade.c
#include <stdio.h>

int main(void){
  printf("Score: ");
  int score;
  scanf("%d", &score);

  if (90 <= score){ printf("Grade: A\n"); }
  else if (80 <= score){ printf("Grade: B\n"); }
  else if (70 <= score){ printf("Grade: C\n"); }
  else if (60 <= score){ printf("Grade: D\n"); }
  else { printf("Grade: F\n"); }
}
